CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,65 @@
+
+* **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+
+    * Violent threats or language directed against another person.
+    * Discriminatory jokes and language.
+    * Posting sexually explicit or violent material.
+    * Posting (or threatening to post) other people's personally identifying information ("doxing").
+    * Personal insults, especially those using racist or sexist terms.
+    * Unwelcome sexual attention.
+    * Advocating for, or encouraging, any of the above behavior.
+    * Repeated harassment of others. In general, if someone asks you to stop, then stop.
+
+* **When we disagree, try to understand why.** Disagreements, both social and technical, happen all the time and ESLint is no exception. It is important that we resolve disagreements and differing views constructively. Remember that we're different. The strength of the ESLintcommunity comes from inclusion of people from a wide range of backgrounds. Different people have different perspectives on issues. Being unable to understand why someone holds a viewpoint doesn't mean that they're wrong. Don't forget that it is human to err and blaming each other doesn't get us anywhere. Instead, focus on helping to resolve issues and learning from mistakes.
+
+
+## Reporting Guidelines
+
+If you believe someone is violating the code of conduct we ask that you report it to the jQuery Foundation by emailing [eslint-conduct@googlegroups.com](mailto:eslint-conduct@googlegroups.com). This is a private mailing list only viewable by project leads and cannot be accessed publicly except to send emails. All reports will be kept confidential. In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate law enforcement first.
+
+In your report please include:
+
+* Your contact info (so we can get in touch with you if we need to follow up).
+* Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+* When and where the incident occurred. Please be as specific as possible.
+* Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link. Screenshots can be useful in the case something is edited or deleted before action is taken.
+* Any extra context you believe existed for the incident.
+* If you believe this incident is ongoing.
+*Any other information you believe we should have.
```

Comment:
Does the bullet need a space? The GitHub renderer at least is rendering it as italic.